pkg/asset/manifests/dns.go

```diff
@@ -152,6 +153,25 @@ func (d *DNS) Generate(dependencies asset.Parents) error {
 			return errors.Wrap(err, "failed ot get DNS zone ID")
```

Comment:
this will require a redo of your lgtm, but, can you fix this typo while you're editing this file? @manojnkumar pointed it out :D

Reply:
Done, force-pushed.